docs/guides/location-service/fragments/setting-up-your-app-android.md

```diff
@@ -0,0 +1,210 @@
+## Overview
+
+[Amazon Location Service](http://aws.amazon.com/location) is a geolocation service that lets you add location information to your applications. With Amazon Location Service, you can build applications that provide maps and points of interest, track resources, trigger actions based on location, and convert street addresses into geographic coordinates. This guide provides you with the following information:
```

Comment:
For a first time reader, who is new to the domain, "assets" may elicit more of a business use case than "resources." My concern with "resources" is that it sounds like an abstract entity within a software system, as opposed to a physical object that a business owns.

```suggestion
[Amazon Location Service](http://aws.amazon.com/location) is a geolocation service that lets you add location information to your applications. With Amazon Location Service, you can build applications that provide maps and points of interest, track assets, trigger actions based on location, and convert street addresses into geographic coordinates. This guide provides you with the following information:
```

Reply:
I tihnk this wordsmithing is great! I'm going to hold it for now since we went over this quite a bit internally, and we can revisit later.